rollup.config.js

```diff
@@ -3,7 +3,7 @@ import json from '@rollup/plugin-json';
 import resolve from '@rollup/plugin-node-resolve';
 import typescript from '@rollup/plugin-typescript';
 import * as pkg from './package.json';
-import { fileURLToPath } from 'node:url';
+// import { fileURLToPath } from 'node:url';
```

Comment:
this comment should be here?

Reply:
Removed: https://github.com/BrainJS/brain.js/commit/b83f6da4db824e3426c898b1a5069eecf139b8f9